packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -335,7 +334,7 @@ class AccountSwitcher extends React.Component {
                                                             account_type={account.group}
                                                             balance={account.balance}
                                                             currency={account.currency}
-                                                            currency_icon={getMT5AccountType(account.group).replace(/^demo/, 'real')}
+                                                            currency_icon={`IcMt5-${getMT5AccountDisplay(account.group)}`}
```

Comment:
Will this work? You could get a string such a "`IcMt5-Synthetic indices`" by using `getMT5AccountDisplay`.

Reply:
@aaron-binary  yup, it will replace empty spaces and low dashes `_` to `-`. In this case, this string will be converted to `ic-mt5-synthetic-indices`